packages/cli/src/command-build.js

```diff
@@ -479,6 +489,8 @@ function makePackageBuild(
   /**@type {import('@bilt/types').Directory}*/ rootDirectory,
   /**@type {{[x: string]: string|boolean | undefined}} */ buildOptions,
   /**@type {object} */ biltin,
+  /**@type {(number)} */ packagesLength,
+  /**@type {(number)} */ pointer,
```

Comment:
I don't see why this needs to be a parameter. It can be a local variable

Reply:
i tried a local variable - but is reset every function call - so I tack the closer approach - do try another way?